### PR TITLE
Bump version to v6.0.0

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
The new ABI version is 13.0.0

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -12,8 +12,76 @@ openssl<sup>1</sup> | >=v1.x
...
+    The definition of the callback function for log handling `drizzle_log_fn`
+    has changed from
+
+    `typedef void (drizzle_log_fn)(const char *line, drizzle_verbose_t verbose,
+    void *context);`
+
+    to
+
+    `typedef void (drizzle_log_fn)(const char *file, uint line, const char *func,
+    const char *msg, drizzle_verbose_t verbose, void *context)`
```
Comment (mathias-lang-sociomantic) [inline RELEASE_NOTES.md:52]: Occurs to me that this could have been made a `struct` for better forward compatibility. Next time ?

**Review by mathias-lang-sociomantic — APPROVED** [review]
LGTM

File: RELEASE_NOTES.md (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply RELEASE_NOTES.md:52]: Yes

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**